README.md

```diff
@@ -1,2 +1,36 @@
-# dst-parser
-This repository is managed by Terraform!
+# Example docstring parser
+
+A source code docstring parser for a custom C-like (including C++, JavaScript, Rust)
+and Python-like syntax, where documentation sections are marked in comments using
+`FRAGMENT` and `ESCAPE` statements.
+
+## Syntax
+
+A text is considered a named fragment when it is located between `// BEGIN FRAGMENT: <name>` and `// END FRAGMENT`, `<>` not included. This syntax is case-sensitive.
+
+Don't use special characters in names; `snakeCase` is preferrable.
+
+Fragments can be included in one another. In that case the lines matching `// BEGIN FRAGMENT: <name>` and `// END FRAGMENT` are removed.
+
+Elements between `// BEGIN ESCAPE` and `// END ESCAPE` are excluded unconditionally from the tutorial.
+
+## Installation
+
+* `npm install dst-parser --save-dev` if you want to install it as a development dependency of your package.
+* `npm install dst-parser -g` to install the parser globally.
```

Comment:
* Install the parser as a development dependency of your package:

  ```
  npm install dst-parser --save-dev
  ```

* Install the parser globally:

  ```
  npm install dst-parser -g
  ```

Reply:
Thanks, added